### ISSUE TITLE
[request] recursively search ancestor groups

### ISSUE THREAD

**Issue by jdmswong** [VeliovGroup/Meteor-flow-router-title#5]
a titlePrefix and default title will affect all routes in a group, but not routes in a child groups group.  It would be great to set these properties for the whole app through a single global group.  Thanks!

**Comment by dr-dimitru** [comment]
Could you please give a little example of how you would like to use it?

**Comment by jdmswong** [comment]
I have a series of routes in groups:

```
ancestorGroup: {
  memberSection: {
    profilePage,
    createPost
  },
  freeContent: {
    freeItem1,
    freeItem2
  }
}
```

I would like to set ancestorGroup.titlePrefix = "Sitename: " to prefix all the titles of routes under it.  So that the title of the profile page would be something like: "Sitename: profile page"; and item1: "Sitename: item1"

**Comment by dr-dimitru** [comment]
Not sure if root routeGroup is accessible in subChild route, I'll check it.
What about global prefix?

**Comment by jdmswong** [comment]
in kadira's flow router you have to recursively travel up the ancestry tree

```
FlowRouter.globals.push({
    titlePrefix: "SiteName - "
});
```

didn't work

**Comment by dr-dimitru** [comment]
Hello @jdmswong ,

Implemented in [v1.2.5](https://github.com/VeliovGroup/Meteor-flow-router-title/releases/tag/2.1.5)
